packages/react-components/src/templates/__tests__/ManuscriptForm.test.tsx

```diff
@@ -574,7 +615,6 @@ describe('renders the necessary fields', () => {
     },
   );
 });
-
 it('resets form fields to default values when no longer visible', async () => {
```

Comment:
I suspect some tests are timing out due to the numerous form interactions involved. Since this test aims to verify that Preprint DOI and Publication DOI are set to undefined when the manuscript lifecycle changes, we can pre-configure the other fields using props. All the quick checks are already pre configured to "Yes" in the `defaultProps`, so we can remove some of the code below. And we can add the `manuscriptFile` to `props` :)


```
it('resets form fields to default values when no longer visible', async () => {
  const onSave = jest.fn();
  render(
    <StaticRouter>
      <ManuscriptForm
        {...defaultProps}
        title="manuscript title"
        onSave={onSave}
        manuscriptFile={{
          id: '123',
          url: 'https://test-url',
          filename: 'abc.jpeg',
        }}
      />
    </StaticRouter>,
  );
```